consumer/src/lib.rs

```diff
@@ -41,6 +42,12 @@ mod tests {
     pub const EMPTY_CONTAINER_3_3_IGNORED_ID: NodeId =
         NodeId(unsafe { NonZeroU64::new_unchecked(13) });
 
+    pub struct NullActionHandler();
```

Comment:
I didn't know that you could declare an empty struct as an empty tuple struct. I'm not a big fan of it though as its initialization syntax makes it look like a method call. You can simply declare it like so:

```rust
pub struct NullActionHandler;
```